examples/playlist/ads.js

```diff
@@ -123,21 +121,18 @@ Ads.prototype.onAdEvent = function(event) {
 };
 
 Ads.prototype.onPlaylistItemClick = function(event) {
-  if (!this.initialized) {
-    this.init();
-  }
   if (!this.linearAdPlaying) {
+    if (!this.initialized) {
+      // Handles the case where the user loads the page and clicks a playlist item
+      // immediately, never clicks the play button on the player.
+      this.init();
+    }
     this.player.ima.setContentWithAdTag(
         this.contents[event.target.id],
         null,
-        true);
```

Comment:
is this change intentional?

Reply:
Yea we now call player.play() in the adsManagerLoaded callback above, so we don't need to content to play on load.